misk/src/test/kotlin/misk/hibernate/RawHibernateApiTest.kt

```diff
@@ -38,21 +37,20 @@ class RawHibernateApiTest {
     }
   }
 
-  // TODO(jwilson): Inject the SessionFactory directly.
-  @Inject @Movies lateinit var sessionFactoryProvider: Provider<SessionFactory>
+  @Inject @Movies lateinit var sessionFactory: SessionFactory
```

Comment:
Does this mean we rebuild the SessionFactory before each test or do we reuse one? I think building a SessionFactory is pretty expensive!

Reply:
The first one takes forever because I think it’s scanning the classpath. Subsequent ones are pretty decent. If necessary I’ll do the stuff to cache things between tests.